crates/yul/benches/parse.rs

```diff
@@ -0,0 +1,72 @@
+use alloy_primitives::U256;
+use criterion::{
+    criterion_group, criterion_main,
+    measurement::{Measurement, WallTime},
+    BenchmarkGroup, Criterion,
+};
+use revive_integration::cases::Contract;
+use revive_yul::{lexer::Lexer, parser::statement::object::Object as AstObject};
+
+/// The function under test parses the Yul `source_code`.
+fn parse(source_code: &str) {
+    let mut lexer = Lexer::new(source_code.to_owned());
+    AstObject::parse(&mut lexer, None).expect("expected a Yul AST Object");
```

Comment:
```suggestion
    AstObject::parse(&mut lexer, None).expect("the Yul source should parse");
```

Please provide either no message (`unwrap()`) or a meaningful message.